craft_providers/multipass/multipass_instance.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Multipass Instance."""
+import io
+import logging
+import os
+import pathlib
+import subprocess
+import sys
+from typing import Any, Dict, List, Optional
+
+from .. import Executor
+from .errors import MultipassError
+from .multipass import Multipass
+
+logger = logging.getLogger(__name__)
+
+
+def _get_host_gid():
+    if sys.platform == "win32":
```

Comment:
can you comment on why we need this and what the implications are of returning 0?

Reply:
That's a good question.  I did it this way because Snapcraft did it!  But I noticed the default value under Windows for me is actually -2...  Need to revisit this.

Reply:
I talked to @townsend2010 and there really isn't mapping support for Windows, so I dropped the mappings entirely on Windows for now